docs/sources/tempo/setup/upgrade.md

```diff
@@ -17,6 +17,79 @@ This upgrade guide applies to on-premise installations and not for Grafana Cloud
 You can check your configuration options using the [`status` API endpoint]({{< relref "../api_docs#status" >}}) in your Tempo installation.
 {{% /admonition %}}
 
+## Upgrade to Tempo 2.5
+
+Tempo 2.5 has several considerations for any upgrade:
+
+* Docker image runs as new UID
+* Support for vParquet format removed
+* Experimental vParquet4 block format
+* Removed configuration parameters
+
+For a complete list of changes, enhancements, and bug fixes, refer to the [Tempo 2.5 changelog](https://github.com/).
+
+### Docker image runs as new UID
+
+The Tempo process in the [official Docker image](https://hub.docker.com/r/grafana/tempo/tags) used to run as `root`. The Tempo process now runs as UID `10001` in the Docker image.
+
+Components such as ingesters and metrics generators that maintain files on disk won't come up cleanly without intervention.
+The new user `10001` won't have access to the old files created by `root`.
+
+The ownership of `/var/tempo` changed from `root:root` to `tempo:tempo` with the UID/GID of `10001`.
+
+The `ingester` and `metrics-generator` statefulsets may need to [run chown](https://opensource.com/article/19/8/linux-chown-command) to change ownership to start properly.
+
+Refer to [PR 2265](https://github.com/grafana/tempo/pull/2265) to see a Jsonnet example of an `init` container.
+
+This change doesn’t impact you if you used the Helm chart with the default security context set in the chart.
+All data should be owned by the `tempo` user already.
+The UID won’t impact Helm chart users.
+
+### Support for vParquet format removed
+
+The original vParquet format [has been removed](https://github.com/grafana/tempo/pull/3663) from Tempo 2.5.
+Direct upgrades from Tempo 2.1 to Tempo 2.5 are not possible.
+You will need to upgrade to an intermediate version and wait for the old vParquet blocks to fall out of retention before upgrading to 2.5. [PR 3663](https://github.com/grafana/tempo/pull/3663)]
+
+vParquet(1) won't be recognized as a valid encoding and can't read any remaining vParquet blocks.
```

Comment:
vParquet(1) won't be recognized as a valid encoding and any remaining vParquet(1) blocks will not be readable

Reply:
```suggestion
vParquet(1) won't be recognized as a valid encoding and any remaining vParquet(1) blocks will not be readable.
```